packages/cli/src/commands/unlink.ts

```diff
@@ -1,9 +1,20 @@
 import unlink from '../scripts/unlink';
 import push from './push';
+import { promptIfNeeded } from '../utils/prompt';
+import ZosPackageFile from '../models/files/ZosPackageFile';
 
 const name: string = 'unlink';
 const signature: string = `${name} [dependencies...]`;
 const description: string = 'unlinks dependencies from the project. Provide a list of whitespace-separated dependency names';
+const props = (depNames) => {
+  return {
+    dependencies: {
+      message: 'Select the dependencies you want to unlink',
+      type: 'checkbox',
+      choices: depNames
```

Comment:
What happens if depNames is empty? We should handle that case nicely

Reply:
Good question. Will check it!